ingestion_tools/scripts/importers/annotation.py

```diff
@@ -119,14 +138,21 @@ def __init__(
         if self.file_format not in ["mrc"]:
             raise NotImplementedError("We only support MRC files for segmentation masks")
 
-    def convert(self, fs: FileSystemApi, input_prefix: str, output_prefix: str, voxel_spacing: float = None):
+    def convert(
+        self,
+        fs: FileSystemApi,
+        input_prefix: str,
+        output_prefix: str,
+        voxel_spacing: float = None,
+        write_mrc: bool = True,
+        write_zarr: bool = True,
+    ):
         input_file = self.get_source_file(fs, input_prefix)
-        return scale_maskfile(
+        return scale_mrcfile(
```

Comment:
Same as above, `write_mrc` and `write_zarr` should probably be passed on to `scale_mrcfile`.